spec/features/posting_peep_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'timecop'
+
+feature 'Posting a new peep' do
+  scenario 'User can post a peep to Chitter' do
+    visit('/')
+    fill_in('message', with: 'I have arrived!')
+    click_button('Submit')
+
+    expect(page).to have_content 'I have arrived!'
+  end
+
+  # scenario 'View the time message was posted' do
```

Comment:
As a first rendition of this test, you could define a timestamp with Time.new, create a new Peep, and then check that the Peep contains the expected information - timestamp.hour, .min, .day, etc. 

It's good practice to use stubs in place of information like Time (which is constantly moving), but it can be easier to implement it with real information first and then refactor as needed.